docs/dev-readme.md

```diff
@@ -0,0 +1,52 @@
+# Developers Guide
+
+## Swagger UI
+
+### View REST API Specification from Swagger Editor UI
+
+Open https://editor.swagger.io/ in a browser. Click `File` - `Import file` from top menu and open the file [$WORKSPACE/redhat-cne/rest-api/v2/swagger.json](../v2/swagger.json). This loads the REST API specification like the following screenshot:
+
+![Alt text](swagger-editor.png "Swagger Editor")
+
+### Interact with REST-API in Swagger UI
+
+You can interact with API endpoint by click `Try it out`, enter required parameters and click `Execute`.
+This requires a REST-API server to be deployed at backend and accessible from localhost.
+
+#### Interact with cloud-event-proxy as the REST-API server
+
+When using cloud-event-proxy sidecar in linuxptp-daemon pod as the REST-API server, the API port in container needs to be port-forwarded to localhost.
+```
+oc port-forward -n openshift-ptp ds/linuxptp-daemon 9043:9043 &
```

Comment:
I am not at all favor of this tainting rest-api project with linuxptp and ptp stuff.  Breaks the whole  architect of decoupled  design

Reply:
it is just an example of backend. I added the words that this is an example.